app/build.gradle

```diff
@@ -1,21 +1,21 @@
 // Build Scripts
-buildscript {
-    repositories {
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
-    }
-    dependencies {
-        classpath "org.jlleitschuh.gradle:ktlint-gradle:11.3.1"
-    }
-}
+//buildscript {
```

Comment:
We should never push commented code. 

You can alway comment stuff out in local commits, but whenever it comes to pushing it to the repo, we should keep the code as clean as possible.

Reply:
Thank you for the feedback. We missed deleting unnecessary comments from gradle files. It has been fixed now. 
[Commit](https://github.com/EON-Incubator/RickAndMorty-Android/commit/e4ba1e4fca490de736803a2acc5f695fcc352600)